### PR TITLE
Change application deadline format

### PR DESCRIPTION
**Pull request by Khristophor**

### CODE REVIEW

File: resources/lang/en/applicant/application_index.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "progress_title" => "Applications in Progress",
     "completed_title" => "Completed Applications",
     "expired_title" => "Expired Applications",
+    "apply_by_label" => "Apply By",
```
[the remark this answers is not in the document]
Comment (gobyrne) [reply resources/lang/en/applicant/application_index.php]: Interestingly the we have a translation for Applicants to Date, but it's not consistent (different on job card then it is on job poster). I'll fix that in this pull request as well.

Apply By = Postuler avant le   (if they localize the date it's possible they add "le" on their own, please make sure it doesn't get duplicated)
Applications to Date = Candidat jusqu'à présent

[the remark this answers is not in the document]
Comment (gobyrne) [reply resources/lang/en/applicant/application_index.php]: and plural is "Candidats" (with an 's')

**Comment by gggrant** [comment]
screen shot 2019-01-21 at 12 17 18 pm

Apply by label should be removed after a job closes.

**Comment by Khristophor** [comment]
@joshdrink the spacing around the dates looks a little weird now, not sure how you want to address
formatted date

**Comment by Khristophor** [comment]
@joshdrink as I mentioned, take a peek at `/fr/applications` with an in-progress application to see what Grant mentioned https://github.com/GCTC-NTGC/TalentCloud/pull/597#pullrequestreview-194686346

File: app/Models/JobPoster.php [hunk 2]
```diff
@@ -64,12 +65,37 @@
...
  */
-class JobPoster extends BaseModel {
+class JobPoster extends BaseModel
+{
 
     use \Dimsav\Translatable\Translatable;
     use Notifiable;
 
-    public $translatedAttributes = ['city', 'title', 'impact', 'branch', 'division', 'education'];
+    const DATE_FORMAT = [
```
Comment (tristan-orourke) [inline app/Models/JobPoster.php]: Date formatting should eventually be moved into a helper class (presumably we'll need to do this elsewhere) but it works for now.

**Pull request merged**